services/blockchain-indexer/shared/indexer/transactionProcessor/interoperability/submitSidechainCrossChainUpdate.js

```diff
@@ -48,22 +55,44 @@ const applyTransaction = async (blockHeader, tx, events, dbTrx) => {
 	};
 
 	await blockchainAppsTable.upsert(appInfo, dbTrx);
+	await ccuTable.upsert({ ...tx, ...tx.params, transactionID: tx.id }, dbTrx);
```

Comment:
Should we specify the params for better readability? 
```suggestion
	await ccuTable.upsert({ height: tx.height, sendingChainID: tx.params.sendingChainID, transactionID: tx.id }, dbTrx);
```